src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -338,6 +340,55 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
         return state_;
     }
 
+    static std::string initialize_directory_(std::string model_name) {
+	    const char* base_directory = std::getenv("VIAM_MODULE_DATA");
+
+	    std::stringstream ss;
+	    ss << base_directory << "/" << model_name << "/1/model.savedmodel";
```

Comment:
This looks tensorflow specific, so it seems contrary to the overall intention of the triton server, which is to be model agnostic.

Could https://viam.atlassian.net/browse/RSDK-8618 be expanded with a bunch more detail about what is being done, and why? Otherwise it is hard to evaluate whether this code review is meeting the goals.

Reply:
We discussed in person: this version was indeed too TF-specific. After talking with Bijan, the plan is that if `saved_model.pb` exists, put in this extra directory, and otherwise symlink the version number directly to the directory where the model data is.